ui/StatusQ/src/StatusQ/Components/StatusCard.qml

```diff
@@ -287,6 +287,13 @@ Rectangle {
             bottomPadding: 0
             leftPadding: 8
             rightPadding: 5
+            input.edit.color: {
```

Comment:
Please add comment with link to the bug. Otherwise somebody can notice inconsistency and restore the previous version unconsciously. Link also helps removing workaround after migration to version without pointed bug.

Reply:
Original code has condition on `input.edit.activeFocus`, shouldn't it be taken into account?

Reply:
Will do but `input.edit.activeFocus` is a simple property, as opposed to `color` which is a grouped prop that poses the problem

Reply:
Fixed

Reply:
> Will do but `input.edit.activeFocus` is a simple property, as opposed to `color` which is a grouped prop that poses the problem

Not sure if I understand. What I mean is that `color` was dependent on `input.edit.activeFocus`
```
input.edit.color: input.edit.activeFocus || !root.locked ?
                                      Theme.palette.directColor1 : Theme.palette.directColor5
```
and I wondered if still should or for some reason it can be skipped.

Reply:
So yeah, can be skipped :)